symplyphysics/laws/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index.py

```diff
@@ -19,11 +18,11 @@
 ## point with the same faces and demonstrate interference.
 
 ## Law is: r = sqrt(k * R * L / n), where
-## r - film thickness,
+## r - radius of Newton's ring,
 ## k - order of interference,
 ## R - radius of curvature of lens
-## L - wavelength,
-## n - refractive index of lens.
+## L - wavelength in medium surrounding lens and plate,
+## n - refractive index of medium between lens and plate.
```

Comment:
As far as I understood, you should either use wavelength in vacuum and adjust it for refractive index of medium, or use wavelength in medium and remove refractive index from formula.